src/gggenome.jl

```diff
@@ -0,0 +1,136 @@
+# GGGenome
+# ======
+#
+# APIs for GGGenome.
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+"""
+GGGenome module.
+The `GGGenome` module provides a programming interface to GGGenome, 
+a ultrafast DNA sequence search service, hosted by DBCLS.
+Two functions are exported from this module:
+1. `gggenome`: DNA sequence search
+2. `gggenome_dblist`: List of available databases
+See "GGGenome Help" (https://gggenome.dbcls.jp/en/help.html) for more details.
+"""
+module GGGenome
+
+export
+    gggsearch,
+    gggdbs
+
+
+import HTTP
+
+const baseURL = "https://gggenome.dbcls.jp/"
+const dblistURL = "https://raw.githubusercontent.com/meso-cacase/GGGenome/master/DBlist.pm"
+
+
+# APIs of GGGenome
+# -------------------
+
+"""
+    gggsearch(query; params...)
+Retrieve results of gggenome search of a query sequence.
+
+# Arguments
+## Required 
+- `query::String`: Nucleotide sequence, case insensitive.
+
+## Optional
+- `db::String`: Target database name. hg19 if not specified. Full list of databases: https://gggenome.dbcls.jp/en/help.html#db_list
+- `k::Integer`: Maximum number of mismatches/gaps. 0 if not specified.
+- `strand::String`: '+' ('plus') or '-' ('minus') to search specified strand only.
+- `format::String`: [html|txt|csv|bed|gff|json]. html if not specified.
+- `timeout::Integer`
+- `output::String`: If "toString", a String object is returned. If "extractTopHit", a String object containing only top hit is returned (Currently, only works with format="txt"). Otherwise, A HTTP.Messages.Response object is returned.
+- `show_url::Bool`: If true, print URL of REST API.
+"""
+function gggsearch(query; timeout=5, params...)
```

Comment:
Maybe I've said somewhere, but I still think you should list optional keyword arguments here like this:
```julia
function gggsearch(query::AbstractString; timeout=5, db="hg19", k=0, strind=nothing, and others...)
```

With this style, developers can grasp the default arguments more quickly without reading the body of this function (and it will be faster than converting arguments to a dictionary, though the speedup would be negligible here).